src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/CaseTatReport.java

```diff
@@ -0,0 +1,206 @@
+              Column.forString("Supplemental Only",
+                  x -> isSupplementalOnly(x.getTest(), x.getCase().getRequisition()) ? "Yes"
+                      : "No"),
+              Column.forString("Extraction Completed Date",
+                  x -> Optional
+                      .ofNullable(findLatestCompletionDate(x.getTest().getExtractions()))
+                      .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse("")),
+              Column.forInteger("Extraction Days", x -> x.getTest().getExtractionDaysSpent()),
+              Column.forString("Library Prep Completed Date",
+                  x -> Optional
+                      .ofNullable(findLatestCompletionDate(x.getTest().getLibraryPreparations()))
+                      .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse("")),
+              Column.forInteger("Library Prep. Days",
+                  x -> x.getTest().getLibraryPreparationDaysSpent()),
+              Column.forString("Library Qual Completed Date",
+                  x -> Optional
+                      .ofNullable(findLatestCompletionDate(x.getTest().getLibraryQualifications()))
+                      .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse("")),
+              Column.forInteger("Library Qual. Days",
+                  x -> x.getTest().getLibraryQualificationDaysSpent()),
+              Column.forString("Full Depth Completed Date",
+                  x -> Optional
+                      .ofNullable(findLatestCompletionDate(x.getTest().getFullDepthSequencings()))
+                      .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse("")),
+              Column.forInteger("Full-Depth Days",
+                  x -> x.getTest().getFullDepthSequencingDaysSpent()),
+              Column.forString("Analysis Review Completed", x -> {
+                List<LocalDate> dates = x.getCase().getDeliverables().stream()
+                    .map(CaseDeliverable::getAnalysisReviewQcDate).collect(Collectors.toList());
+                if (dates.contains(null)) {
+                  return null;
+                }
+                return dates.stream().max(LocalDate::compareTo)
+                    .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse(null);
+              }),
+              Column.forInteger("Analysis Review Days",
+                  x -> x.getCase().getAnalysisReviewDaysSpent()),
+              Column.forString("Release Approval Completed", x -> {
+                List<LocalDate> dates = x.getCase().getDeliverables().stream()
+                    .map(CaseDeliverable::getReleaseApprovalQcDate).collect(Collectors.toList());
+                if (dates.contains(null)) {
+                  return null;
+                }
+                return dates.stream().max(LocalDate::compareTo)
+                    .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse(null);
```

Comment:
This is identical to "Analysis Review Completed" except for the map parameter. Can extract to a shared method